neqo-udp/src/lib.rs

```diff
@@ -268,6 +268,8 @@ mod tests {
             .expect("receive to succeed");
 
         // Assert that the ECN is correct.
+        // FIXME: This test currently fails in the Android emulator.
+        #[cfg(not(target_os = "android"))]
         assert_eq!(
             IpTosEcn::from(datagram.tos()),
             IpTosEcn::from(received_datagrams.next().unwrap().tos())
```

Comment:
@mxinden do we know if ECN w/`quinn-udp` is supposed to work on Android?

Reply:
Unfortunately, it depends on the Android API Level:

https://github.com/quinn-rs/quinn/pull/2079

Reply:
Good idea. Let me add that.